deploy/file_list

```diff
@@ -1,8 +1,10 @@
-Assets/Plugins/Android/SoomlaAndroidCore.jar
```

Comment:
@borisspektor, I checked your suggestion. It works fine with Core module, also it works fine with Store module BUT:
1) It copied all jars/iOS-libs including files which will be populated from other package places, such as `AndroidProfileFacebook.jar` or `AndroidStoreGooglePlay.jar` (is user will test module from repo and then decide to build it into unitypackage;
2) It copies Core libraries but doesn't copies Core scripts and user can receive a lot of errors. We can make core module embedded in other modules (maybe it's implemented already, but in this feature branch scripts creates separate packages), but we should discuss it before implementing.